packages/components/src/mobile/keyboard-aware-flat-list/index.ios.js

```diff
@@ -29,14 +29,54 @@ export const KeyboardAwareFlatList = ( {
 	onScroll,
 	...listProps
 } ) => {
+	const scrollViewRef = useRef();
+	const keyboardWillShowIndicator = useRef();
+
 	const latestContentOffsetY = useSharedValue( -1 );
+
 	const scrollHandler = useAnimatedScrollHandler( {
 		onScroll: ( event ) => {
 			const { contentOffset } = event;
 			latestContentOffsetY.value = contentOffset.y;
 			onScroll( event );
 		},
 	} );
+
+	const getRef = useCallback(
+		( ref ) => {
+			scrollViewRef.current = ref;
+			innerRef( ref );
+		},
+		[ scrollViewRef, innerRef ]
```

Comment:
I think that's not really necessary because we're accessing the reference (i.e. `scrollViewRef.current`). Although the function is memoized, while `scrollViewRef doesn't change, the reference retrieved by `current` property will have the latest value.

Actually, I'm thinking that most likely we even don't need to add the references (i.e. `scrollViewRef`) in the dependency list, since [React guarantees their inmutability](https://reactjs.org/docs/hooks-reference.html#useref). I'll remove `scrollViewRef` from here and do the same on the rest of the callbacks.